### DISCUSSION TITLE
Update backup_archives.list - to be able to restore on older cloned vm

### DISCUSSION THREAD

**Discussion by elephantastyczny** [nextcloud/all-in-one#4023] [Questions] [answered]
Hi Guys,

Once again I need your help.
My VM hosting docker with AIO just died, but I made a clone of it like 2 weeks ago.

I can start Mastercontainer in order to restore the backup, but latest one that I can see is done 2 weeks ago.

How I can update backup_archives.list file in order to show all backups taken between 16.12.2023 and today?

20230630_040108-nextcloud-aio,2023-06-30 04:01:10
20230731_040120-nextcloud-aio,2023-07-31 04:01:22
20230831_040101-nextcloud-aio,2023-08-31 04:01:03
20230930_040059-nextcloud-aio,2023-09-30 04:01:01
20231031_040111-nextcloud-aio,2023-10-31 04:01:13
20231112_040050-nextcloud-aio,2023-11-12 04:00:52
20231119_040059-nextcloud-aio,2023-11-19 04:01:01
20231126_040038-nextcloud-aio,2023-11-26 04:00:40
20231130_214535-nextcloud-aio,2023-11-30 21:45:37
20231203_040045-nextcloud-aio,2023-12-03 04:00:47
20231210_040043-nextcloud-aio,2023-12-10 04:00:46
20231211_040056-nextcloud-aio,2023-12-11 04:00:58
20231212_040057-nextcloud-aio,2023-12-12 04:01:00
20231213_040108-nextcloud-aio,2023-12-13 04:01:10
20231213_211717-nextcloud-aio,2023-12-13 21:17:19
20231214_040055-nextcloud-aio,2023-12-14 04:00:57
20231215_040046-nextcloud-aio,2023-12-15 04:00:48
20231216_040114-nextcloud-aio,2023-12-16 04:01:16

**Comment by szaimen** [comment, marked as answer]
Hi, in this doc, there is a hint how to update the backup archives list: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#delete-backup-archives-manually
